apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -355,6 +356,17 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
             }
         }
 
+        if (selectedTemplate.id === ApplicationTemplateIdTypes.M2M_APPLICATION) {
+            // M2M Apps are created with `client_credentials` grant by default.
+            application.inboundProtocolConfiguration = {
+                oidc: {
+                    grantTypes: [
+                        "client_credentials"
```

Comment:
Is it possible to use an already define constant here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4605